### ISSUE TITLE
Feature request: Enable an option for Constant rate of turning.

### ISSUE THREAD

**Issue by DMJC** [vegastrike/Vega-Strike-Engine-Source#729]
Wing Commander: Privateer ships use a constant rate of turning e.g 45 degree/sec for the Tarsus with no inertial effects on the turning. To make Privateer mods feel more like the original game I would like the Vegastrike engine to be updated to add an optional flag which would allow a constant turn rate to be used, rather than a mass/acceleration/time formula.

**Comment by BenjamenMeyer** [comment]
@Loki1950 is that handled by the different physics parameters?

**Comment by Loki1950** [comment]
Yes it's one of the terms in the physics frame differential equations.

**Comment by stephengtuggy** [comment]
OK, sounds like we need to implement this / get it working again. Perhaps the only question is when.

**Comment by stephengtuggy** [comment]
@BenjamenMeyer what board should this go on?

**Comment by BenjamenMeyer** [comment]
This should be on the engine board. We may have to do some deep dive research to figure out how it would work. We may be a couple releases out before we can really address it though based on current conversation.

**Comment by stephengtuggy** [comment]
I see. What I was specifically asking about is which project board should it go on. Probably should have been more clear.

**Comment by BenjamenMeyer** [comment]
> I see. What I was specifically asking about is which project board should it go on. Probably should have been more clear.

no problem - I created a new VS Engine board yesterday so it should definitely go on there; while we hash things out it should probably also be on the VS Proposals board; and I added the new Intake board for the new process.

**Comment by evertvorster** [comment]
My 0.02c:

Vegastrike used to work like this, and I have explained many times how the physics behind this actually works. 

What VS does now is _wrong_ , and "constant rate of turn" should be the default (and only) option.

In short, your space ship has a flight computer, and when it is in "MANEUVER" mode, that flight computer is limits your ship speed to the set speed, and does not allow it exceed the set speed. 

It achieves this by firing retro-rockets and side rockets to keep your speed constant through turns. 

Think of it as "cruise control" for your ship. At your set speed, your ship is NOT accelerating to stay at the set speed (graphics non-withstanding) Any flight input you do is almost all on your orthogonal thrusters. Your much more powerful retro and forward thrusters are not used at their full power when you alter course, unless you manage to do a 180 and have your big thruster pointing in the opposite way that you are traveling, and then your speed will go down before it goes up again. 

In travel mode, your set speed are a few orders of magnitude higher, and this has very little effect because you almost never reach your set speed anyways.

**Comment by BenjamenMeyer** [comment]
@evertvorster one of my favorite things with WC VI was to use a game modifier to change the physics settings to realistic where you would slide through turns - turns were never constant speed then; it made for some interesting flight dynamics but also a lot of fun.

That said, simplistic flight characteristics should be the default for ease of game play.

**Comment by evertvorster** [comment]
You absolutely can do that with VS as well, there is a very sparsely documented key, I think the ~ key, that, when pressed turns off side thrusters, and allows you to "slide"

https://wiki.vega-strike.org/Manual:Keyboard_layout:US

Look for Shelton Key
